ext/BreezeCloudMicrophysicsExt/one_moment_microphysics.jl

```diff
@@ -387,17 +386,18 @@ The rate follows Morrison and Milbrandt (2015, JAS), Eq. (A1):
 Evaporation is limited to the available cloud liquid to prevent negative values.
 
 # References
-- Morrison, H. and Milbrandt, J.A. (2015), J. Atmos. Sci., 72, 287-311.
-  https://doi.org/10.1175/JAS-D-14-0065.1
+* Morrison, H., and J. A. Milbrandt (2015). Parameterization of cloud microphysics based on
+    the prediction of bulk ice particle properties. Part I: Scheme description and idealized
+    tests. J. Atmos. Sci., 72, 287–311.
```

Comment:
Same comment about the DOI as in https://github.com/NumericalEarth/Breeze.jl/pull/345#discussion_r2645468381